IntegrationExample/AppDelegate.swift

```diff
@@ -9,27 +9,55 @@ import Foundation
 import MimiCoreKit
 import MimiUXKit
 import MimiSDK
+import Combine
 
-final class AppDelegate: NSObject, UIApplicationDelegate {
+final class AppDelegate: NSObject, UIApplicationDelegate, ObservableObject {
```

Comment:
You can move `ObservableObject` to the `PartnerHeadphoneConnectivityController` and inject it as an `environmentObject`.